README.md

```diff
@@ -151,64 +125,22 @@ Build the supervisor with a specific tag, and for a specific architecture, like
-set there. Keep in mind several images will be pulled for caching purposes - if the base image is not cached the build
-can take a few hours.
+set there. Keep in mind several images will be pulled for caching purposes.
 
-The Docker caching system can be tricky, so it might try to build the base image anyways; if you see that the build hangs at something like:
+## Base image
 
-```
-### Shell environment set up for builds. ###
-
-You can now run 'bitbake <target>'
-
-Common targets are:
-    core-image-minimal
-    core-image-sato
-    meta-toolchain
-    meta-ide-support
-
-You can also run generated qemu images with a command like 'runqemu qemux86'
-```
-
-...don't be alarmed, it means it is building the base image and will take a few hours without any output. Go grab some coffee or play a game of Risk.
-
-## Working with the Go supervisor
-
-The code for the Go supervisor lives in the `gosuper` directory.
-
-To build it, run (with the ARCH and IMAGE you want):
-```bash
-make ARCH=amd64 IMAGE=username/gosuper:master gosuper
-```
-This will build a Docker image that builds the Go supervisor and has the executable at /go/bin/gosuper inside the image.
-
-### Adding Go dependencies
-This project uses [glide](https://github.com/Masterminds/glide) to manage its Go dependencies. Refer to its repository for instructions on adding packages.
-
-In order for go utilities to work, this repo needs to be within the `src` directory in a valid Go workspace. This can easily be achieved by having the repo as a child of a directory named `src` and setting the `GOPATH` environment variable to such directory's parent.
+The supervisor uses the [resin-supervisor-base](https://github.com/resin-io/resin-supervisor-base) as a base image.
+This is a minimal Linux image containing busybox, rsync and Node.js, and it's built with the [Yocto project](https://www.yoctoproject.org/).
 
 ## Testing
 
-We're working on adding more tests to this repo, but here's what you can run in the meantime:
+You can run some unit tests with:
 
-### Gosuper
-
-The Go supervisor can be tested by running:
-```bash
-make ARCH=amd64 test-gosuper
 ```
-The test suite is at [gosuper/main_test.go](./gosuper/main_test.go).
-
-### Integration test
-The integration test tests the supervisor API by hitting its endpoints. To run it, first run the supervisor as explained in the first section of this document.
-
-Once it's running, you can run the test with:
-```bash
-make ARCH=amd64 test-integration
+npm test
 ```
 
-The tests will fail if the supervisor API is down - bear in mind that the supervisor image takes a while to start the actual supervisor program, so you might have to wait a few minutes between running the supervisor and testing it.
-The test expects the supervisor to be already running the application (so that the app is already on the SQLite database), so check the dashboard to see if the app has already downloaded.
+You'll need Node.js 6 installed, and having run `npm install` first.
```

Comment:
"You'll need Node.js 6 installed and to have run `npm install` first." 

Is Node.js 6 an exact or minimum requirement?

Reply:
Exact - I've clarified this a bit more.